src/screens/SendToken/SendTokenContacts.js

```diff
@@ -465,6 +470,28 @@ class SendTokenContacts extends React.Component<Props, State> {
     return [{ title: `Send ${tokenName}` }];
   };
 
+  renderFooter = () => {
+    const { value, isValidatingEns } = this.state;
+    const isSearchQueryProvided = !!(value && value.address.length);
+    const submitDisabled = !value.address.length || isValidatingEns;
+    return (
+      <>
+        {isSearchQueryProvided && (
+          <ButtonWrapper>
+            <Button
+              height={48}
+              regularText
+              textStyle={fontStyles.medium}
+              disabled={submitDisabled}
+              title="Next"
+              onPress={this.handleFormSubmit}
+            />
+          </ButtonWrapper>
+        )}
+      </>
+    );
```

Comment:
```suggestion
    if (!isSearchQueryProvided) return null;
    return (
      <ButtonWrapper>
        <Button
          height={48}
          regularText
          textStyle={fontStyles.medium}
          disabled={submitDisabled}
          title="Next"
          onPress={this.handleFormSubmit}
        />
      </ButtonWrapper>
    );
```

Reply:
I tried this at first and this doesn't do the job. But it does when you replace `null` with `<></>`, so committing that. RN never fails to surprise me...